addons/escoria-dialog-simple/types/floating.gd

```diff
@@ -142,10 +161,12 @@ func say(character: String, line: String) :
 		.format([text_color_html]) + tr(line) + "[/color][center]"
 
 	if _current_character.is_inside_tree() and \
-			_current_character.has_node("dialog_position"):
-		rect_position = _current_character.get_node(
-			"dialog_position"
-		).get_global_transform_with_canvas().origin
+			dialog_location_node != null:
+#		rect_position = _current_character.get_node(
```

Comment:
Is this commented block still needed?

Reply:
Oops, thanks! Fixed. Also found another comment I'd left in and fixed that too.